packages/fxa-auth-server/lib/routes/recovery-phone.ts

```diff
@@ -56,11 +68,18 @@ class RecoveryPhoneHandler {
   }
 
   async setupPhoneNumber(request: AuthRequest) {
-    const { uid } = request.auth.credentials as SessionTokenAuthCredential;
+    const { uid, email } = request.auth
+      .credentials as SessionTokenAuthCredential;
+
     const { phoneNumber } = request.payload as unknown as {
       phoneNumber: string;
     };
 
+    if (!email || !uid) {
+      throw AppError.invalidToken();
+    }
+    await this.customs.check(request, email || '', 'recoveryPhoneCreate');
```

Comment:
Why the ` || ''`, especially when line 78 above is checking for an email?

Reply:
So, there was a bad mock in a test. Once I fixed the mock, the code of course started working. I think I threw that in as a temp workaround and forgot to take it out. Thanks for catching.